app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -16,28 +16,20 @@ app.views.Notifications = Backbone.View.extend({
     var note = $(evt.target).closest(".stream_element");
     var unread = note.hasClass("unread");
 
-    if (unread) {
-      this.setRead(note.data("guid"));
-    }
-    else {
-      this.setUnread(note.data("guid"));
-    }
+    if (unread){ this.setRead(note.data("guid")); }
+    else { this.setUnread(note.data("guid")); }
   },
 
-  setRead: function(guid) {
-    $.ajax({
-      url: "/notifications/" + guid,
-      data: { set_unread: false },
-      type: "PUT",
-      context: this,
-      success: this.clickSuccess
-    });
-  },
+  getAllUnread: function(){ return $('.media.stream_element.unread'); },
```

Comment:
Neither did we about camel case and 'Diaspora' is defined in the .jshint.json. I have no idea what @houndci is doing here.